src/netius/clients/dht.py

```
@@ -315,7 +315,9 @@ class DHTClient(netius.DatagramClient):
     """
 
     def ping(self, host, port, peer_id, *args, **kwargs):
-        return self.query(type="ping", *args, **kwargs)
+        return self.query(
+            host=host, port=port, peer_id=peer_id, type="ping", *args, **kwargs
```

Ruff (0.16.2)

[warning] Star-arg unpacking after a keyword argument is strongly discouraged

(B026)